transport/src/main/java/io/netty/channel/SingleThreadEventLoop.java

```diff
@@ -148,6 +148,14 @@ public int pendingTasks() {
         return super.pendingTasks() + tailTasks.size();
     }
 
+    /**
+     * Returns the number of {@link Channel}s registered with this {@link EventLoop} or {@code -1}
+     * if operation is not supported.
```

Comment:
@vkostyukov please just add some sentence that the number may be not 100 % correct.

Reply:
Also please tag the method with `@UnstableAPI`

Reply:
Done. Let me know what do you think.